.github/workflows/erlang.yml

```diff
@@ -0,0 +1,35 @@
+name: Erlang CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: erlang:22.0.7
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Compile
+      run: rebar3 compile
+    - name: Check formatting
+      run: DEBUG=1 rebar3 steamroll --check
```

Comment:
Why DEBUG=1 still?

Reply:
strangely format check is failing on github actions and working locally and on travis. I was checking if I could get more information, but was not helpful